src/services/crmanagers/OpenshiftResourceManagerFactory.js

```diff
@@ -0,0 +1,12 @@
+import { GenericResurceManager } from './GenericResurceManager';
+import { PushVariantResourceManager } from './PushVariantResourceManager';
+
+export class OpenshiftResourceManagerFactory {
```

Comment:
I believe this could simply be called `ResourceManagerFactory`. What do you think?